apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -64,9 +70,14 @@ const Instructions: React.FunctionComponent<InstructionsProps> = props => {
   return (
     <InstructionsPanel
       text={instructionsText}
-      message={validationState.message || undefined}
+      message={message || undefined}
       showNextButton={showNextButton}
-      onNextPanel={() => dispatch(navigateToNextLevel())}
+      onNextPanel={() => {
```

Comment:
nit: what do you think abobut wrapping this with `useCallback`?

Reply:
Done!